tests/integration/snapshots/logs/async-metrics_python38_with_plugin.log

```diff
@@ -2,23 +2,23 @@ START RequestId: XXXX Version: $LATEST
 END RequestId: XXXX
 REPORT RequestId: XXXX	Duration: XXXX ms	Billed Duration: XXXX ms	Memory Size: 1024 MB	Max Memory Used: XXXX MB	
 XRAY TraceId: XXXX	SegmentId: XXXX	Sampled: true	
 START RequestId: XXXX Version: $LATEST
 {"m": "aws.lambda.enhanced.invocations", "v": 1, "e": XXXX, "t": ["region:us-east-1", "account_id:XXXX", "functionname:integration-plugin-dev-async-metrics_python38_with_plugin", "resource:integration-plugin-dev-async-metrics_python38_with_plugin", "cold_start:false", "memorysize:1024", "runtime:python3.8", "dd_lambda_layer:datadog-python38_2.XX.0"]}
 {"m": "hello.dog", "v": 1, "e": XXXX, "t": ["team:serverless", "role:hello", "dd_lambda_layer:datadog-python38_2.XX.0"]}
 {"m": "tests.integration.count", "v": 21, "e": XXXX, "t": ["test:integration", "role:hello", "dd_lambda_layer:datadog-python38_2.XX.0"]}
-{"traces": [[{"trace_id": "XXXX", "parent_id": "XXXX", "span_id": "XXXX", "service": "aws.lambda", "resource": "integration-plugin-dev-async-metrics_python38_with_plugin", "name": "aws.lambda", "error": 0, "start": XXXX, "duration": XXXX, "meta": {"_dd.origin": "lambda", "runtime-id": "XXXX", "cold_start": "false", "function_arn": "arn:aws:lambda:us-east-1:601427279990:function:integration-plugin-dev-async-metrics_python38_with_plugin", "function_version": "$LATEST", "request_id": "XXXX", "resource_names": "integration-plugin-dev-async-metrics_python38_with_plugin", "_dd.parent_source": "xray"}, "metrics": {"system.pid": XXXX, "_sampling_priority_v1": 2}, "type": "serverless"}]]}
+{"traces": [[{"trace_id": "XXXX", "parent_id": "XXXX", "span_id": "XXXX", "service": "aws.lambda", "resource": "integration-plugin-dev-async-metrics_python38_with_plugin", "name": "aws.lambda", "error": 0, "start": XXXX, "duration": XXXX, "meta": {"_dd.origin": "lambda", "runtime-id": "XXXX", "cold_start": "false", "function_arn": "arn:aws:lambda:us-east-1:601427279990:function:integration-plugin-dev-async-metrics_python38_with_plugin", "function_version": "$LATEST", "request_id": "XXXX", "resource_names": "integration-plugin-dev-async-metrics_python38_with_plugin", "datadog_lambda": "2.23.0", "dd_trace": "0.41.2", "_dd.parent_source": "xray"}, "metrics": {"system.pid": XXXX, "_sampling_priority_v1": 2}, "type": "serverless"}]]}
```

Comment:
Seems like we should filter out the version numbers and replace them with "XXXX" so that the snapshots don't change every time we change the version number.

Reply:
Yeah, agreed, we should probably mask them. I believe @meghna-dd added these very recently.

Reply:
Done!